rai/node/bootstrap.cpp

```diff
@@ -1032,8 +1032,8 @@ void rai::bootstrap_attempt::requeue_pull (rai::pull_info const & pull_a)
 	}
 	else if (pull.attempts == 4)
 	{
+		pull.attempts++;
```

Comment:
Not specific to this PR, but it seems to me that pull_info#attempts should be atomic.